core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -263,13 +263,18 @@ coreos:
       command: start
       content: |
         [Unit]
-        Requires={{.Etcd.SystemdUnitName}}
+        Wants={{.Etcd.SystemdUnitName}}
         After={{.Etcd.SystemdUnitName}}
 
         [Service]
-        Type=oneshot
+        Type=simple
+        Restart=on-failure
+        RestartSec=10
+
         EnvironmentFile={{.EtcdNodeEnvFileName}}
-        ExecStart=/opt/bin/cfn-signal
+        ExecStartPre=/usr/bin/systemctl is-active {{.Etcd.SystemdUnitName}}
+        ExecStartPre=/usr/bin/rkt fetch --pull-policy new {{.AWSCliImage.Options}}{{.AWSCliImage.RktRepo}}
+        ExecStart=-/opt/bin/cfn-signal
```

Comment:
Ah, and I realized we can keep the minus here regardless of we deal with the temporary cfn-signal failure - the retry I've mentioned should be implemented inside /opt/bin/cfn-signal rather than being handled by systemd.
Thanks for the clarification 👍